{"62b43e6912a64c8eaa3d9e86da5a604e":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"d572fe9fbb7673ad","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/24//#testresult/d572fe9fbb7673ad","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55866afc2b13 <unknown>\n#1 0x55866adc9688 <unknown>\n#2 0x55866adedf1f <unknown>\n#3 0x55866ade95aa <unknown>\n#4 0x55866ae2464a <unknown>\n#5 0x55866ae1e7a3 <unknown>\n#6 0x55866adf40ea <unknown>\n#7 0x55866adf5225 <unknown>\n#8 0x55866b00a2dd <unknown>\n#9 0x55866b00e2c7 <unknown>\n#10 0x55866aff422e <unknown>\n#11 0x55866b00f0a8 <unknown>\n#12 0x55866afe8bc0 <unknown>\n#13 0x55866b02b6c8 <unknown>\n#14 0x55866b02b848 <unknown>\n#15 0x55866b045c0d <unknown>\n#16 0x7f5b71edf609 <unknown>","time":{"start":1657831041449,"stop":1657831042960,"duration":1511}},{"uid":"40eb5480234ec27e","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/23//#testresult/40eb5480234ec27e","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55e1825d2b13 <unknown>\n#1 0x55e1823d9688 <unknown>\n#2 0x55e1823fdf1f <unknown>\n#3 0x55e1823f95aa <unknown>\n#4 0x55e18243464a <unknown>\n#5 0x55e18242e7a3 <unknown>\n#6 0x55e1824040ea <unknown>\n#7 0x55e182405225 <unknown>\n#8 0x55e18261a2dd <unknown>\n#9 0x55e18261e2c7 <unknown>\n#10 0x55e18260422e <unknown>\n#11 0x55e18261f0a8 <unknown>\n#12 0x55e1825f8bc0 <unknown>\n#13 0x55e18263b6c8 <unknown>\n#14 0x55e18263b848 <unknown>\n#15 0x55e182655c0d <unknown>\n#16 0x7fb79cd5c609 <unknown>","time":{"start":1657830368832,"stop":1657830372761,"duration":3929}}]},"4db9caae8ef7326232c323be69d7c921":{"statistic":{"failed":5,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":5},"items":[{"uid":"beec529e99322af5","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/24//#testresult/beec529e99322af5","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657831041419,"stop":1657831041419,"duration":0}},{"uid":"3cf05b2daf80b506","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/23//#testresult/3cf05b2daf80b506","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657830368801,"stop":1657830368802,"duration":1}},{"uid":"d9f492d8c5490526","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/22//#testresult/d9f492d8c5490526","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657742529712,"stop":1657742529712,"duration":0}},{"uid":"4b4a9418cbf6ae3b","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/20//#testresult/4b4a9418cbf6ae3b","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657741951648,"stop":1657741951648,"duration":0}},{"uid":"6c3cb921775d6672","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/19//#testresult/6c3cb921775d6672","status":"failed","statusDetails":"AssertionError: assert 2 == 3","time":{"start":1657730708932,"stop":1657730708933,"duration":1}}]},"d89e5d3dcae2f873575c0ac61689d19c":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"ae56afc0d717c14","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/25//#testresult/ae56afc0d717c14","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657838605942,"stop":1657838605943,"duration":1}}]},"2de973a726f6108f7c3b207b90ba2fc4":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"ae55eb0012aa2a06","reportUrl":"https://javiermayan.github.io/test-automation-python-practice/25//#testresult/ae55eb0012aa2a06","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://chromedriver.chromium.org/home","time":{"start":1657838605845,"stop":1657838605846,"duration":1}}]}}